UI/entrypoint.sh

```diff
@@ -8,7 +8,7 @@ set -e
 
 mkdir -p /opt/app-root/src/config
 
-cat <<EOF > /opt/app-root/src/config/config.js
```

Comment:
that's a clear bug, I don't know how it is working now :)

The point of adding this config file was to make the backend API URL configurable. The react app generates pages and settings during build time and so it was not straightforward to have a runtime environment variable for the backend API. I'm open to suggestions / alternatives to this approach, my understanding here is limited. 

We'd have to test any changes by rebuilding the image and testing the deployment on OpenShift. We need Github Actions.

Reply:
makes sense!